### PR TITLE
min: update to v1.22.1

### PR DESCRIPTION
**Pull request by ryanfortner**

### CODE REVIEW

File: apps/Min/install-32 [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-install_packages https://github.com/ryanfortner/minbrowser-arm-builds/releases/download/v1.22/min_1.22.0_armhf.deb || exit 1
+install_packages https://github.com/ryanfortner/minbrowser-arm-builds/releases/download/1.22.1/min_1.22.1_armhf.deb || exit 1
```
Comment (Itai-Nelken) [inline apps/Min/install-32:3]: Is the `v` removed intentionally?

Comment (ryanfortner) [reply apps/Min/install-32:3]: Yes.

File: apps/Min/install-64 [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-install_packages https://github.com/minbrowser/min/releases/download/v1.22.0/min_1.22.0_arm64.deb || exit 1
+install_packages https://github.com/minbrowser/min/releases/download/v1.22.1/min_1.22.1_arm64.deb || exit 1
```
Comment (Itai-Nelken) [inline apps/Min/install-64:3]: Can youbremove the `v` here as well to be consistent please?

Comment (Itai-Nelken) [reply apps/Min/install-64:3]: oh, I didn't see, never mind.

**Comment by Itai-Nelken** [comment]
Thanks!

**Pull request merged**